internal/v2/fakes2av2/fakes2av2.go

```diff
@@ -98,15 +99,19 @@ func offloadPrivateKeyOperation(req *s2av2pb.OffloadPrivateKeyOperationReq, host
 			s := fmt.Sprintf("Invalid hostname tied to SessionReq: %s", hostname)
 			return nil, errors.New(s)
 		}
-
-		// TODO(rmehta19): Parse req.GetSignatureAlgorithm() to find Hash function that
-		// should be used here. Currently hardcoded to use SHA256.
-
-		// Sign the SHA256 hash with the private key.
-		var opts crypto.Hash = crypto.SHA256
-		signedBytes, err := root.PrivateKey.(crypto.Signer).Sign(rand.Reader, req.GetInBytes(), opts)
-		if err != nil {
-			return nil, err
+		var signedBytes []byte
+		if req.GetSignatureAlgorithm() == s2av2pb.SignatureAlgorithm_S2A_SSL_SIGN_RSA_PKCS1_SHA256 {
+			var opts crypto.Hash = crypto.SHA256
```

Comment:
readability nit: I think this line should just be `opts := crypto.SHA256`

In fact, should we just omit the `opts` local variable altogether?

Reply:
Sounds good! I'll pass crypto.SHA256 directly to the Sign function then.